pkg/kv/kvserver/gc/gc.go

```diff
@@ -116,6 +131,16 @@ var MaxIntentKeyBytesPerCleanupBatch = settings.RegisterIntSetting(
 	settings.NonNegativeInt,
 )
 
+// ClearRangeMinKeys is a minimum number of keys that GC will consider
+// for executing a ClearRange request. Since ClearRange requests translate into
+// pebble tombstones, they should be used on large sequence of keys only.
+var ClearRangeMinKeys = settings.RegisterIntSetting(settings.SystemOnly,
+	"kv.gc.clear_range_min_keys",
```

Comment:
I was thinking that it should be like a way to turn if off if it affects storage performance of a bug is discovered. For memory usage, we don't have direct visibility so having memory setting questionable. Maybe we need a budget with metric in that case. Having a setting is also tempting because I don't know what to expect.

Reply:
Ok, makes sense.